places-solution-integrations/target/attach-data.md

```diff
@@ -0,0 +1,76 @@
+# Attaching Places Data to Target Events via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
+
+Once Places is sending in events for entries and exits, you can leverage Rules in Launch to attach your Places data to your Adobe Target SDK events. With your desired property selected in Launch, you can create this type of rule by completing four steps:
```

Comment:
change to:
"After Places sends entry and exit events, you can leverage Rules in Launch to attach your Places data to your Adobe Target SDK events. To create this type of rule, select your property in Launch, and complete the following steps: